src/components/RemainingTime.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const RemainingTime = ( {remainingSeconds} ) => {
-    const minutes = Math.floor(remainingSeconds/60);
-    const seconds = remainingSeconds % 60;
-    return(
-        <Time>
-            {minutes} : {seconds}
-        </Time>
-    )
-}
+const RemainingTime = ({ remainingSeconds, breakTime }) => {
+  const minutes = Math.floor(remainingSeconds / 60);
+  const seconds = remainingSeconds % 60;
+  return (
+    <div>
+        <h2>{breakTime ? "BREAK" : "SESSION"}</h2>
```

Comment:
Pls move this out of the `RemainingTime` component as the component doesn't really have to do with anything else than remaining time.

Reply:
... also, the name `breakTime` is misleading. The convention for boolean vars is to have some sort of `is` prefix. In this case `breakTime` could be taken to represent a time value.